package.json

```diff
@@ -80,6 +80,5 @@
   },
   "engines": {
     "node": ">= 16.0.0"
-  },
```

Comment:
![Screenshot 2024-11-28 at 14 55 06](https://github.com/user-attachments/assets/738ac4c7-16c0-45fc-8a42-6fcf25c4f47b)

That restriction should have been removed